DuckDuckGo/BookmarkCell.swift

```diff
@@ -52,9 +52,13 @@ class BookmarkCell: UITableViewCell {
 
         if let domain = domain {
             let faviconUrl = AppUrls().faviconUrl(forDomain: domain)
+
             linkImage.kf.setImage(with: faviconUrl,
                                   placeholder: placeholder,
-                                  options: [.downloader(NotFoundCachingDownloader())],
+                                  options: [
+                                    .downloader(NotFoundCachingDownloader()),
+                                    .targetCache(ImageCache(name: "bookmarks"))
```

Comment:
Maybe the same should be used for `FavoriteHomeCell`?

Reply:
🤦‍♂  thanks